README.md

```diff
@@ -55,6 +55,32 @@ config :file_storage_api, :azure_blob,
 
 It will need the package `file` in linux to be able to read mime types and work.
 
+## Dynamic configuration
+
+You can use a map to pass a long dynamic configuration in the connection field.
+
+For azure
+```elixir
+%{
+  engine: :azure,
+  config: %{
+    host: "postfix.docker",
+    secret_key: "YWNjb3VudF9rZXk=",
+    access_key: "amazing"
+  }
+}
+```
+
+For S3
+```elixir
+%{
+  engine: :s3,
+  config: %{host: "test.docker", secret_key: "test123", access_key: "amazing", scheme: "http://"}
```

Comment:
maybe you can do test.example instead of .docker

Reply:
improved the example